test/e2e/gpu/gpu.go

```diff
@@ -98,7 +98,7 @@ func describe() {
 		framework.ExpectNoError(err, "pod Create API error")
 
 		ginkgo.By("waiting the pod to finish successfully")
-		e2epod.NewPodClient(f).WaitForSuccess(ctx, pod.ObjectMeta.Name, 60*time.Second)
+		errPodNotSuccess := e2epod.WaitForPodSuccessInNamespace(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name)
```

Comment:
What's the timeout after the change?

Reply:
sorry, I didn't know there is 'WaitForPodSuccessInNamespaceTimeout'. I used and put the same timeout as before.!